lib/Bio/EnsEMBL/Tark/Hive/PipeConfig/HGNCName_conf.pm

```diff
@@ -0,0 +1,84 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+=head1 DESCRIPTION
+
+  A pipeline for the loading of the HGNC gene names into a Tark DB.
+
+=head1 SYNOPSIS
+
+  # Prepare the pipeline
+  init_pipeline.pl Bio::EnsEMBL::Tark::Hive::PipeConfig::HGNCName_conf
+    --tark_host $TARK_HOST
+    --tark_port $TARK_PORT
+    --tark_user $TARK_USER
+    --tark_pass $TARK_PASS
+    --tark_db $TARK_SPP_DB
+    --host $HIVE_HOST
+    --port $HIVE_PORT
+    --user $HIVE_USER
+    --password $HIVE_PASS
+    --pipeline_name test_hive_1234
+    --hgnc_file $HGNC_NAME_FILE
+
+=cut
+
+package Bio::EnsEMBL::Tark::Hive::PipeConfig::HGNCName_conf;
+
+use strict;
+use warnings;
+
+# All Hive databases configuration files should inherit from HiveGeneric, directly
+# or indirectly
+use base ('Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf');
+
+# Allow this particular config to use conditional dataflow and INPUT_PLUS
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
```

Comment:
This weirded me out as well. Turned out I needed to import as well to get some of the functionality working

Reply:
IIRC, it is because INPUT_PLUS isn't defined in HiveGeneric_conf.pm, but it is exported by it. It's actually defined in Bio/EnsEMBL/Hive/Utils/PCL.pm.
When you `use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf`, Exporter puts INPUT_PLUS into your namespace.